spec/std/complex_spec.cr

```diff
@@ -169,6 +169,11 @@ describe "Complex" do
     c.clone.should eq(c)
   end
 
+  it "hashes real without imag like real only" do
```

Comment:
typo: `imag` => `image`

Or maybe not (you did the same in the code below) ? Isn't it `image` in english?

Reply:
image is known shortcut for Imaginary part of complex number.

Reply:
ok thanks, I didn't know